pyramid_swagger/tween.py

```diff
@@ -542,10 +542,10 @@ def validate_response(response, validator_map):
 
 
 def prepare_body(response):
-    # content_type and charset must both be set to access response.text
-    if response.content_type is None or response.charset is None:
+    # content_type must be set to access response.text
+    if not response.content_type:
```

Comment:
@sjaensch, @striglia: what will happen if someone uses pyramid_swagger with ``webob<1.7``?
I think that it will produces errors if response.charset is not defined, so I would enforce ``webob>=1.7`` in ``setup.py``